packages/gatsby/src/utils/page-data.ts

```diff
@@ -108,7 +110,8 @@ export async function writePageData(
     },
   })
 
-  await fs.outputFile(outputFilePath, bodyStr)
+  // await fs.outputFile(outputFilePath, bodyStr)
+  await ldmbStore.put(outputFilePath, bodyStr)
```

Comment:
In theory we can also skip `JSON.stringify` step here + add `sharedStructuresKey` option to `lmdb-store` get some benefits of `msgpackr` vs `json`

```js
let myStore = open('my-store', {
	sharedStructuresKey: Symbol.for('structures')
})
```

Reply:
💯 all the speeds